### ISSUE TITLE
Gramformer on pypi?

### ISSUE THREAD

**Issue by hangyav** [PrithivirajDamodaran/Gramformer#35]
Hi,

Thanks for the great tool. I'm working on a language server for grammatical error detection (https://github.com/hangyav/textLSP) and included Gramformer as one option for text analysis. However, I'm not able to include it in the dependencies, since pypi does not accept github links in ```setup.py```, thus it needs to be installed separately. I was wondering if you are planning to release Gramformer on pypi? Thanks!

**Comment by FellaBennadji** [comment]
Hi,
Could u install it ? I'm facing the same problem with setup.py

**Comment by hangyav** [comment]
Yes, I can install it manually using pip:

```pip install git+https://github.com/PrithivirajDamodaran/Gramformer.git```

**Comment by FellaBennadji** [comment]
with Windows ?

**Comment by hangyav** [comment]
No, I only tried on Linux.

**Comment by FellaBennadji** [comment]
Yeah, it seem not working on Windows. I used Ubuntu to install it. Thank you for the answer.
